charts/kong/example-values/README.md

```diff
@@ -14,6 +14,11 @@ change this value to `true` if you use Helm 2).
 * [minimal-k4k8s-enterprise.yaml](minimal-k4k8s-enterprise.yaml) installs Kong
   for Kubernetes Enterprise with the ingress controller in DB-less mode.
 
+* [minimal-kong-enterprise-dbless.yaml](minimal-kong-enterprise-dbless.yaml)
```

Comment:
```suggestion
* [minimal-kong-enterprise-dbless.yaml](minimal-kong-enterprise-dbless.yaml) **beta release**
```

Reply:
Different change; bolded in the prose below as IMO it's important to state what that means in context, since we haven't established it here otherwise (hopefully people have read the docs site explanation of beta stuff, but especially as our Enterprise beta stuff is new for 2.1, I figure we should err on the side of repeating what we mean by that).